.gitignore

```diff
@@ -1,7 +1,9 @@
 *.pyc
 *.pyo
-plugincheck/settings_local.py
+plugincheck/settings/local.py
+plugincheck/settings/test.py
```

Comment:
oh hmm. I was torn because mine has my db username/password. I was thinking of writing the docs to have them copy local.py.dist into test.py?